packages/unhead/src/client/renderDOMHead.ts

```
@@ -22,25 +22,21 @@ function _renderDOMHead<T extends Unhead<any>>(head: T, options: RenderDomHeadOp
     return false
   const beforeRenderCtx: DomBeforeRenderCtx = { shouldRender: true, tags: [] }
   callHook(head, 'dom:beforeRender', beforeRenderCtx)
-  if (!beforeRenderCtx.shouldRender || head._du)
-    return false
+  if (!beforeRenderCtx.shouldRender || head._du) return false
```

ESLint

[error] Expect newline after if

(antfu/if-newline)